cmp/grid/GridModel.js

```diff
@@ -640,6 +638,20 @@ export class GridModel {
         const Model = XH.isMobile ? MobileColChooserModel : DesktopColChooserModel;
         return this.markManaged(new Model(this));
     }
+
+    defaultContextMenuFn = (agParams, gridModel) => {
+        if (XH.isMobile) return null;
+        return new StoreContextMenu({
+            items: GridModel.defaultContextMenuTokens,
+            gridModel
```

Comment:
The impl in grid calls the ctx menu fn with the grid model as the second argument, so I am using that here vs. relying on `this` as it had been.....